source/admin_manual/list_of_events.rst

```diff
@@ -2726,6 +2726,91 @@ Event emitted when dict server finishes transaction. Same fields as
 :ref:`dict_transaction_finished`.
 
 
+Login proxy
+===========
+
+.. versionadded:: v2.3.18
+
+Emitted when login processes proxy a user connection to a backend.
+
+.. _pre_login_client:
+
+Pre-login Client Common fields
+------------------------------
+
++---------------------+------------------------------------------------------+
+| Field               | Description                                          |
++=====================+======================================================+
+| local_ip            | Local IP address                                     |
++---------------------+------------------------------------------------------+
+| local_port          | Local port                                           |
++---------------------+------------------------------------------------------+
+| remote_ip           | Remote IP address                                    |
++---------------------+------------------------------------------------------+
+| remote_port         | Remote port                                          |
++---------------------+------------------------------------------------------+
+| user                | Full username                                        |
++---------------------+------------------------------------------------------+
+
+Proxy Login Common Fields
+-------------------------
```

Comment:
This can then be just "Common fields" ("fields" is lowercase elsewhere currently at least)